ui/tests/acceptance/secrets/backend/azure/azure-configuration-test.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { click, visit, currentURL } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { v4 as uuidv4 } from 'uuid';
+import { spy } from 'sinon';
+
+import authPage from 'vault/tests/pages/auth';
+import enablePage from 'vault/tests/pages/settings/mount-secret-backend';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { runCmd } from 'vault/tests/helpers/commands';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { overrideResponse } from 'vault/tests/helpers/stubs';
+import { SECRET_ENGINE_SELECTORS as SES } from 'vault/tests/helpers/secret-engine/secret-engine-selectors';
+import { mountBackend } from 'vault/tests/helpers/components/mount-backend-form-helpers';
+import {
+  expectedConfigKeys,
+  expectedValueOfConfigKeys,
+  configUrl,
+} from 'vault/tests/helpers/secret-engine/secret-engine-helpers';
+
+module('Acceptance | Azure | configuration', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function () {
+    const flash = this.owner.lookup('service:flash-messages');
+    this.store = this.owner.lookup('service:store');
+    this.flashSuccessSpy = spy(flash, 'success');
+    this.flashInfoSpy = spy(flash, 'info');
+    this.version = this.owner.lookup('service:version');
+    this.uid = uuidv4();
+    return authPage.login();
+  });
+  module('isEnterprise', function (hooks) {
+    hooks.beforeEach(function () {
+      this.version.type = 'enterprise';
+    });
+
+    test('it should show empty state and navigate to configuration view after mounting the azure engine', async function (assert) {
+      const path = `azure-${this.uid}`;
+      await visit('/vault/settings/mount-secret-backend');
+      await mountBackend('azure', path);
+
+      assert.dom(GENERAL.emptyStateTitle).hasText('Azure not configured');
+      assert.dom(GENERAL.emptyStateActions).doesNotContainText('Configure Azure');
+      // cleanup
+      await runCmd(`delete sys/mounts/${path}`);
+    });
+
+    test('it should not show "Configure" from toolbar', async function (assert) {
+      const path = `azure-${this.uid}`;
+      await enablePage.enable('azure', path);
+      assert.dom(SES.configure).doesNotExist('Configure button does not exist.');
+      assert.strictEqual(
+        currentURL(),
+        `/vault/secrets/${path}/configuration`,
+        'navigated to configuration view'
+      );
```

Comment:
It looks like this assertion should actually be in the test case above. Since both of these assertions are related to the empty state view I think they can all be combined into one test. And perhaps the configure button assertion could just be more explicit `Configure toolbar action does not exist`

Reply:
That does make more sense, thank you.